storage/innobase/xtrabackup/src/keyring_components.cc

```diff
@@ -122,6 +120,33 @@ void create_component_config_data() {
       std::string(string_buffer.GetString(), string_buffer.GetSize());
 }
 
+bool read_server_uuid() {
+  if (xtrabackup_stats) return true;
```

Comment:
whats special about stats? :)

Reply:
Stats its been my nightmare :D . It can be run against a live server / offline server / backup. For the first two, we don't have backup-my.cnf

Reply:
Please add a comment before the return true :)